From ec35b5d3a4e51bb4455887c1066e8f0696210df2 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 10 Dec 2022 19:57:03 +0100
Subject: [PATCH] com.webos.service.configurator.perm.json: Add permissions for
 callbacks

So the callbacks from our legacy components keep working.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 files/sysbus/com.webos.service.configurator.perm.json | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.configurator.perm.json b/files/sysbus/com.webos.service.configurator.perm.json
index d82d0bc..f098220 100644
--- a/files/sysbus/com.webos.service.configurator.perm.json
+++ b/files/sysbus/com.webos.service.configurator.perm.json
@@ -12,6 +12,14 @@
         "keymgrdatabase.management",
         "tempdatabase.management",
         "mediadatabase.management",
-        "database.management"
+        "database.management",
+        "application.launcher",
+        "calendarremindersservice.operation",
+        "contactslinkerservice.operation",
+        "contactsservice.operation",
+        "imlibpurple-service.operation",
+        "messaging-service.operation",
+        "telephony.management",
+        "update-service.operation"
     ]
 }
-- 
2.37.3.windows.1

